ci/assembly-split-release-artifacts.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+RELEASE_TAG=$1
+SOURCE_DIR=$2
+OUTPUT_DIR=$3
+
+mkdir -p $OUTPUT_DIR/github
```

Comment:
Potentially minor nitpick, but: should this script know about the `github` part of the path? At the moment that segment is the distinction between private and public artefacts, and from that perspective I'm thinking it may be better for that bit to be handled by the YAML file.

Reply:
Our `.sh` files already know about it e.g. `copy-unix-release-artifacts.sh` so keeping it here seems consistent.